src/main/java/frc/robot/RobotContainer.java

```diff
@@ -92,6 +111,22 @@ public RobotContainer() {
     initializeTrajectories();
   }
 
+  private void controllerCheck() {
```

Comment:
FYI DriverStation has an [isJoystickConnected](https://first.wpi.edu/wpilib/allwpilib/docs/release/java/src-html/edu/wpi/first/wpilibj/DriverStation.html#line.911)​(int stick) method. Seems to check the same things though, so this code is fine as is.